packages/lesswrong/components/comments/CommentsItem/CommentsItem.jsx

```diff
@@ -191,16 +195,18 @@ class CommentsItem extends Component {
               }
               <div className="comments-item-date">
                 { this.props.frontPage ?
-                  <Link to={Posts.getPageUrl(this.props.post) + "#" + comment._id}>
+                  <Link to={Posts.getPageUrl(post) + "#" + comment._id}>
                     {moment(new Date(comment.postedAt)).fromNow()}
                     <FontIcon className="material-icons comments-item-permalink"> link
                     </FontIcon>
+                    <span className={classes.postTitle}>{ showPostTitle && post && post.title } </span>
                   </Link>
                 :
-                <a href={Posts.getPageUrl(this.props.post) + "#" + comment._id} onClick={this.handleLinkClick}>
+                <a href={Posts.getPageUrl(post) + "#" + comment._id} onClick={this.handleLinkClick}>
                   {moment(new Date(comment.postedAt)).fromNow()}
                   <FontIcon className="material-icons comments-item-permalink"> link
                   </FontIcon>
+                  <span className={classes.postTitle}> { showPostTitle && post && post.title }</span>
```

Comment:
Perhaps we should just not show this span at all if there is no title.

Reply:
Yeah, that seems correct